src/bin/orval.ts

```diff
@@ -19,7 +19,11 @@ program
   .option('-p, --project <name>', 'focus a project of the config')
   .action((paths, cmd) => {
     if (isString(cmd.input) && isString(cmd.output)) {
-      generateSpec(process.cwd(), { input: cmd.input, output: cmd.output });
+      generateSpec(process.cwd(), {
+        input: cmd.input,
+        output: cmd.output,
+        converterOptions: undefined,
```

Comment:
Only converterOptions  from command line is not supported in PR.

Reply:
I would have put the converterOptions inside the input options